__tests__/app.test.js

```diff
@@ -222,4 +228,97 @@ describe('app', () => {
+            .then((response) => {
+                const result = response.body.comments;
+                expect(Array.isArray(result)).toBe(true);                
+                expect(result).toHaveLength(11);
+                result.forEach((comment) => {
+                    expect(typeof comment).toBe('object');
+                })
+             })
+        })
+        test('status: 200 and responds with a nested array containing all of the comment objects with the correct keys for a specific article_id', () => {
+            return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then((response) => {
+                const result = response.body.comments;
+                result.forEach((comment) => {
+                    expect(comment).toHaveProperty("comment_id");
+                    expect(comment).toHaveProperty("votes");
+                    expect(comment).toHaveProperty("created_at");
+                    expect(comment).toHaveProperty("author");
+                    expect(comment).toHaveProperty("body");
+                    expect(comment).toHaveProperty("article_id");
+                })
+             })
+        })
+        test('status: 200 and responds with a nested array containing all of the comment objects for a specific article_id sorted in descending date order by default', () => {
+            return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then((response) => {
+                const result = response.body.comments;
+                const dates = result.map((comment) => {
+                    return comment.created_at;
+                })
+                expect(dates).toBeSorted({descending: true});
+            })
+        })
+        test('status: 400 when passed a bad article_id', () => {
+            return request(app)
+                .get('/api/articles/dog/comments')
+                .expect(400)
+                .then((response) => {
+                    const msg = response.body.msg;
+                    expect(msg).toBe('Invalid article_id');
+                })
+        })
+        test('status: 404 when passed an article_id that doesnt exist in the database', () => {
+            return request(app)
+                .get('/api/articles/99999/comments')
+                .expect(404)
+                .then((response) => {
+                    const msg = response.body.msg;
+                    expect(msg).toBe('article_id does not exist');
+                })
+        })
+        test('status: 404 when a comment does not exist for a specific article_id', () => {
+            return request(app)
+                .get('/api/articles/2/comments')
+                .expect(404)
+                .then((response) => {
+                    const msg = response.body.msg;
+                    expect(msg).toBe('There are no comments for the specified article_id');
+                })
+        })
```

Comment:
iunclude a test for an invalid id being passed through the endpiont. ie. `/api/articles/not-a-valid-format-for-an-id/comments`